README.md

```diff
@@ -43,9 +47,38 @@ Consider the following fictitious scenario:
 
 Akiko is going away for the weekend. Her good friend Boris is going to borrow her car while she's away. They meet at a nearby cafe, and Akiko hands Boris her car keys. Boris now has the capability to drive Akiko's car whenever he wants to. Depending on their plans for the rest of the day, Akiko may find Boris quite rude if he immediately leaves the cafe to go for a drive. On the other hand, if Akiko asks Boris to run some last minute pre-vacation errands for that require a car, she may expect Boris to immediately drive off.
 
+To put this in terms closer to a UCAN flow:
+
+``` mermaid
+sequenceDiagram
+    participant 🚗
+    actor Akiko
+    actor Boris
+
+    autonumber
+
+    Note over 🚗, Akiko: Akiko buys a car
+    🚗 -->> Akiko: Delegate(Drive 🚗)
+
+    Note over Akiko, Boris: Boris offers to run errands for Akiko
+    Boris -->> Akiko: Delegate(Boris to run errands)
+
+    Note over Akiko, Boris: Akiko gives Boris access to her car
+    Akiko -->> Boris: Delegate(Drive 🚗)
+
+    Note over 🚗, Boris: Akiko asks Boris to use her car to run errands
+    Akiko ->> Boris: Invoke!(Boris to run errands, using 🚗 (➌))
+    Boris ->> 🚗: Invoke!(Drive 🚗)
+```
+
+In the example above, steps ➌ and ➍ are qualitatively different:
+
+- Step ➌ grants authority (to drive the car)
+- Step ➍ is a _command_ to do so
```

Comment:
```suggestion
- Step ➍ is a _request_ to do so
```
To not have name collision with `Command` of this spec

Reply:
Or maybe
- Step ➌ authorizes capability (to drive the car)
- Step ➍ requests to _excercise_ it